packages/perseus/src/widgets/interactive-graphs/graphs/polygon.tsx

```diff
@@ -116,9 +117,16 @@ export const PolygonGraph = (props: Props) => {
                     key={"point-" + i}
                     snapTo={snapTo}
                     point={point}
-                    onMove={(destination: vec.Vector2) =>
-                        dispatch(actions.polygon.movePoint(i, destination))
-                    }
+                    onMove={(destination: vec.Vector2) => {
+                        const now = Date.now();
+                        const targetFPS = 40;
```

Comment:
What's the rationale for capping FPS at 40? For displays that refresh at 60Hz (or browsers that repaint at 60Hz?), this will result in every third frame getting dropped. I guess my question is, why not 30 fps?

Reply:
Loosely held opinion: 40 fps feels about right. I've noticed in games it's where things start to feel smooth (which is totally subjective). Some data, though: If I cap at 40fps I get around 26% CPU usage. If I cap at 60 I get around 40%.